commenter.js

```diff
@@ -15,19 +15,19 @@ var config = require('./config');
  * https://developer.github.com/v3/activity/events/types/#pullrequestevent
  */
 function postPullRequestComment(commentURL, comment, filename, commitSHA, line, done) {
-    sendRequest({
+
+    request.post({
         url: commentURL,
-        method: 'POST',
         headers: {
             'User-Agent': config.brand,
             'Authorization': 'token ' + config.token
         },
-        body: JSON.stringify({
+        form: {
```

Comment:
At the moment, we manually send a request to post the comment. At some point in the future, octonode should implement `create_pull_request_comment`. At that point, we'll have the option of making our code a little cleaner by [getting rid of this file](https://github.com/darkwing/Discord/blob/test-recorder/commenter.js#L3-L6) and using their implementation instead. When that happens, will we have other ways of testing this?